### PR TITLE
Renamed columns for clarification

### PR DESCRIPTION
**Pull request by jennifertran**
Based on issue #75 
 
Renamed the following columns for clarification:
    - `Plasmid Genes` to `Plasmid` in Summary table.
    - `Gene` to `Plasmid` in PlasmidFinder table.
    - `Gene` to `Gene/Plasmid` in Detailed Summary table.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Version 0.5.1
```
Comment (apetkau) [inline CHANGELOG.md]: Awesome :+1:. Could you also update the version in <https://github.com/phac-nml/staramr/blob/master/staramr/__init__.py>?

File: README.md [hunk 2]
```diff
@@ -234,7 +234,7 @@ There are 7 different output files produced by `staramr`:
 2. `detailed_summary.tsv`: A detailed summary of all detected AMR genes/mutations/plasmids in each genome, one gene per line.
 3. `resfinder.tsv`: A tabular file of each AMR gene and additional BLAST information from the **ResFinder** database, one gene per line.
 4. `pointfinder.tsv`: A tabular file of each AMR point mutation and additional BLAST information from the **PointFinder** database, one gene per line.
-5. `plasmidfinder.tsv`: A tabular file of each AMR plasmid gene and additional BLAST information from the **PlasmidFinder** database, one plasmid gene per line.
+5. `plasmidfinder.tsv`: A tabular file of each AMR plasmid type and additional BLAST information from the **PlasmidFinder** database, one plasmid gene per line.
```
Comment (apetkau) [inline README.md]: Change **one plasmid *gene* per line** => **one plasmid *type* per line**

File: README.md [hunk 3]
```diff
@@ -328,7 +329,7 @@ The **pointfinder.tsv** output file generated by `staramr` contains the followin
 The **plasmidfinder.tsv** output file generated by `staramr` contains the following columns:
 
 * __Isolate ID__: The id of the isolate/genome file(s) passed to `staramr`.
-* __Gene__: The particular plasmid gene detected.
+* __Plasmid__: The particular plasmid gene detected.
```
Comment (apetkau) [inline README.md]: Can you change **The particular plasmid *gene* detected.** => **The particular plasmid *type* detected.**.

Also, for the `pointfinder.tsv` section of this document (above) there's some extra **()**:

> * Gene: The particular AMR gene detected, with the point mutation within **()**.

File: doc/tutorial/staramr-tutorial.ipynb [hunk 4]
```diff
@@ -54,35 +54,35 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "--2019-04-10 15:17:07--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/478/105/GCF_001478105.1_Salmonella_enterica_CVM_N31384-SQ_v1.0/GCF_001478105.1_Salmonella_enterica_CVM_N31384-SQ_v1.0_genomic.fna.gz\n",
+      "--2019-05-07 12:17:37--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/478/105/GCF_001478105.1_Salmonella_enterica_CVM_N31384-SQ_v1.0/GCF_001478105.1_Salmonella_enterica_CVM_N31384-SQ_v1.0_genomic.fna.gz\n",
```
Comment (apetkau) [inline doc/tutorial/staramr-tutorial.ipynb]: Thanks so much for taking the time to update this file :smile: 

It looks like for the `plasmidfinder.tsv` and `summary_detailed.tsv` files the older **Gene** column names are still displayed:

image

File: CHANGELOG.md (hunk 1 above)
Comment (jennifertran) [reply CHANGELOG.md]: Fixed in commit 22b711b

File: README.md (hunk 2 above)
Comment (jennifertran) [reply README.md]: Resolved in commit 22b711b

File: README.md (hunk 3 above)
Comment (jennifertran) [reply README.md]: Removed in 22b711b

File: doc/tutorial/staramr-tutorial.ipynb (hunk 4 above)
Comment (jennifertran) [reply doc/tutorial/staramr-tutorial.ipynb]: That's strange, thought I already updated when I first ran through it, maybe not. Changed it in 22b711b

**Comment by jennifertran** [comment]
> Thanks so much @jennifertran, this is awesome . Exactly what I wanted.
> 
> In addition to the comments below, I noticed the `README.md` file is missing some of the newer output files in the table of contents:
> 
> image
> 
> Could you add in a link to `summary_detailed.tsv` and `plasmidfinder.tsv` here?

Yeah I fixed that in commit 22b711b. Thanks for pointing that out.

File: README.md [hunk 5]
```diff
@@ -63,8 +63,10 @@ staramr search -o out --pointfinder-organism salmonella *.fasta
   * [Dependencies](#dependencies)
 - [Output](#output)
   * [summary.tsv](#summarytsv)
+  * [detailed_summary.tsv](#detailedsummarytsv)
```
Comment (apetkau) [inline README.md]: I believe the link should be to `(#detailed_summarytsv)`

File: doc/tutorial/staramr-tutorial.ipynb (hunk 4 above)
Comment (apetkau) [reply doc/tutorial/staramr-tutorial.ipynb]: Thanks :smile:. Those changes all look good.

Note that there's a few places where the text **plasmid genes** appears. Could you change them to **plasmid type**?

Comment (jennifertran) [reply doc/tutorial/staramr-tutorial.ipynb]: Sure will do, fixed that in 061fdf8

File: README.md (hunk 5 above)
Comment (jennifertran) [reply README.md]: Whoops, I thought that would have worked too, apparently not, fixed that in 0ab9cf2

**Pull request merged**